driverdataaggregation/README.md

```diff
@@ -0,0 +1,404 @@
+        client tasks per compute node for the synthetic scaling app
+    --client_nodes=CLIENT_NODES
+      Default: [128, 256, 512]
+        number of compute nodes to use for the synthetic scaling app
+    --iterations=ITERATIONS
+        Default: 20
+        number of append/retrieve loops run by the applications
+    --tensor_bytes=TENSOR_BYTES
+        Default: [1024, 8192, 16384, 32769, 65538, 131076, 262152, 524304, 10...
+        list of tensor sizes in bytes
+    --tensors_per_dataset=TENSORS_PER_DATASET
+        Default: [1, 2, 4]
+        list of number of tensors per dataset
+    --client_threads=CLIENT_THREADS
+        Default: [1, 2, 4, 8, 16, 32]
+        list of the number of client threads used for data aggregation
+    --cpu_hyperthreads==CPU_HYPERTHREADS
+        Default: 2
+        the number of hyperthreads per cpu.  This is done
+                                 to request that the consumer application utilizes
+                                 all physical cores for each client thread.
+    --languages=LANGUAGES
+        Default: ['cpp']
+        list of languages to use for the tester "cpp" and/or "fortran"
+    --wall_time=WALL_TIME
+        Default: '05:00:00'
+        allotted time for database launcher to run
+    --plot=PLOT
+        Default: 'database_nodes'
+        flag to plot against in process results
+```
+For example, the following command could be run to execute a battery of
+tests in the same allocation. The battery of test will be determined by the number 
+of permutations computed based on the list inputs.
+
+```bash
+python driver.py aggregation_scaling --client_nodes=[60] \
+                                     --clients_per_node=[48] \
+                                     --db_nodes=[16,32] \
+                                     --db_cpus=32 --net_ifname="ipogif0" \
+                                     --run_db_as_batch=False \
+                                     --tensors_per_dataset=[1,4]
+```
+
+This command can be executed in a terminal with an interactive allocation
+or used in a batch script such as the following for Slurm based systems
+
+```bash
+#!/bin/bash
+
+#SBATCH -N 93
+#SBATCH --exclusive
+#SBATCH -t 12:00:00
+#SBATCH -C SK48
+#SBATCH --oversubscribe
+
+cd ..
+module load slurm
+python driver.py aggregation_scaling --client_nodes=[60] \
+                                     --clients_per_node=[48] \
+                                     --db_nodes=[16,32] \
+                                     --db_cpus=32 --net_ifname="ipogif0" \
+                                     --run_db_as_batch=False \
+                                     --tensors_per_dataset=[1,4]
+```
+
+Examples of batch scripts to use are provided in the ``batch_scripts`` directory
+
+
+## Data Aggregation Standard - Python Client
+
+The ``aggregation_scaling_python`` test uses a Python client and a SmartRedis Orchestrator. Depending on your user application, you may want to run the scaling test with a python client.
```

Comment:
Again, this is not quite a useful statement. Could be something like "The SmartRedis client is built on top of the C++ SmartRedis layer and thus may behave slightly differently than the C++ client. Differences between this test and the previous C++-based test may ..."

Reply:
wait why would someone want to run data agg with a python client instead of C++?

Reply:
If they're doing some kind of in situ analysis/visualization, they are likely going to be using Python